homework7

```diff
@@ -0,0 +1,30 @@
+Caleb Gershengorn
+
+1:
+++*p:advances the pointer one space in the array in memory, than checks what is in that spot.
+*p++: checks the space in memory at that spot, then advances one space.
+*++p" does the same as the first, because the placement of the '++' makes the difference, not the '*'
+
+2:
+Left to right garuntees operator precedence.
```

Comment:
The simple answer to this question is neither. C doesn’t always evaluate left-to-right or right-to-left. Generally, function calls are evaluated first, followed by complex expressions and then simple expressions.